Sources/TuistGenerator/Generator/BuildPhaseGenerator.swift

```diff
@@ -469,8 +469,11 @@ final class BuildPhaseGenerator: BuildPhaseGenerating {
 
         copySwiftMacrosBuildPhase.inputPaths = executableNames.map { "$BUILD_DIR/$CONFIGURATION/\($0)" }
 
-        copySwiftMacrosBuildPhase.outputPaths = executableNames.map { executable in
-            "$BUILD_DIR/Debug-$EFFECTIVE_PLATFORM_NAME/\(executable)"
+        copySwiftMacrosBuildPhase.outputPaths = executableNames.flatMap { executable in
+            [
+                "$BUILD_DIR/Debug$EFFECTIVE_PLATFORM_NAME/\(executable)",
```

Comment:
it turns out this output path _is_ correct for macos builds but not other platforms. We need to keep both for sandboxing to work on all platforms